### ISSUE TITLE
Stuck on `1.5.9`?

### ISSUE THREAD

**Issue by fosskers** [fukamachi/qlot#307]
## Describe the bug

I've lost confidence over which version of `qlot` I'm actually running. I swear I'm building `1.6.0` from the `.tar.gz`, but somehow when I run `qlot --version`, I see:

```
colin@yumi ~/c/a/qlot (master)> qlot --version
Qlot 1.5.9
colin@yumi ~/c/a/qlot (master) [255]> which qlot
/usr/bin/qlot
colin@yumi ~/c/a/qlot (master)> aura -Qo qlot
/usr/bin/qlot is owned by qlot 1.6.0-1
```

Do you know what might be happening here? Thank you.

## Reproducible steps

I am installing Qlot from https://aur.archlinux.org/packages/qlot , but manually bumping the version to `1.6.0`. Here is how it builds it: https://aur.archlinux.org/cgit/aur.git/tree/PKGBUILD?h=qlot

## Versions:
 - Qlot version: ???
 - Where did you get it from: AUR
 - OS: Arch Linux
 - Lisp: 2.4.9

**Comment by fosskers** [comment]
I think stale caches under `~/.cache/common-lisp/` might have been the problem. I have cleared everything in there involving qlot (even from older sbcl versions!) and the messages of the build process seem different now.

**Comment by fosskers** [comment]
No, unfortunately it's still saying `1.5.9`.

**Comment by fosskers** [comment]
Wild theory: when `qlot` is running, it is somehow not finding certain library files it needs under `/usr/lib/qlot/`, but is instead traversing my filesystem in a strange way and finding an older version of itself inside the `.qlot/` of one of my Common Lisp projects. Perhaps https://github.com/fukamachi/qlot/issues/269 is related.

**Comment by fosskers** [comment]
Indeed: I just deleted the `.qlot` under my copy of `ultralisp`, and now `qlot --version` reports

**Comment by daninus14** [comment]
What worked for me was to run the uninstaller and install again. Are you still having this issue?

**Comment by svetlyak40wt** [comment]
> I no longer use qlot, so we can close this.

@fosskers  what do you use now instead of qlot?
